src/ploneintranet/activitystream/browser/activity_provider.py

```diff
@@ -135,7 +136,8 @@ def Creator(self):
     @property
     def text(self):
         url = api.portal.get().absolute_url()
-        text = link_users(self.context.text, url)
+        text = safe_unicode(self.context.text)
+        text = link_users(text, url)
```

Comment:
This might seem strange, but it's only an interim stage. With the changes in the fix-mentions branch (yet to be PRed and merged), creating the text to show in the status will be streamlined.